eve/main.yml

```diff
@@ -70,18 +70,22 @@ stages:
           usePTY: true
           haltOnFailure: true
       - ShellCommand:
-          name: Put the iso file in a separate folder
+          name: Put the artifacts to upload in a separate folder
           command: >
-            mkdir iso &&
-            cp _build/metalk8s.iso _build/SHA256SUM _build/root/product.txt iso
+            mkdir artifacts &&
+            cp build.log artifacts &&
```

Comment:
Yeah but it's YAML stuff, I prefer keep things simple here :p

Wasted too many time with the weird YAML syntax rules, the line continuations, …

Let's hope we will get rid of this unreadable mess this has become since the addition of upgrade/downgrade, and replace it by some code that generate it :D